src/System.Threading.Thread/tests/System.Threading.Thread.Tests.csproj

```diff
@@ -15,8 +15,10 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Unix_Release|AnyCPU'" />
   <ItemGroup>
+    <Compile Include="CompressedStackTests.cs" />
     <Compile Include="ExceptionTests.cs" />
     <Compile Include="ThreadExceptionEventArgsTests.cs" />
+    <Compile Include="ThreadTests.netstandard1.7.cs" />
```

Comment:
In that case I would probably keep it as ThreadTests.cs for now, and just rename when the time comes for adding tests for Thread targeting older TFMs